spotbugs/src/main/java/edu/umd/cs/findbugs/charsets/UTF8.java

```diff
@@ -47,11 +48,6 @@
      *
      */
     private static final String UTF_8 = "UTF-8";
```

Comment:
Could also drop this and use `StandardCharsets.UTF_8.name`

Reply:
Oh, Thanks!

Reply:
Fixed! 😄